frontend/src/components/WorkOrderModal.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react'
-import { Modal, View, Button } from 'react-native';
+import { Modal, View, Button, Platform } from 'react-native';
 import { styles, buttons } from '../stylesheets/Stylesheet';
 
 const WorkOrderModal = (props) => (
     <Modal visible={props.display} transparent={true} animationType="fade"
         onRequestClose={() => props.closeModal()} >
-        <View style={{ justifyContent: 'center', borderWidth: 2, borderColor: 'black', marginVertical: '25%', marginHorizontal: '5%', height: '75%', width: '90%', backgroundColor: 'rgba(52, 52, 52, 0.8)' }}>
-                <View style={buttons.buttonSignUp}>
-                    <Button onPress={() => props.closeModal()} title="close"></Button>
-                </View>
-        </View>
+        <SafeAreaView style={{ flex: 1, paddingTop: (Platform.OS === "android" || Platform.OS === "ios") ? StatusBar.currentHeight : 0 }}> 
+            <View style={{ justifyContent: 'center', borderWidth: 2, borderColor: 'black', marginVertical: '25%', marginHorizontal: '5%', height: '75%', width: '90%', backgroundColor: 'rgba(52, 52, 52, 0.8)' }}>
```

Comment:
pls put it in a stylesheet file and import the styles here

Reply:
This file was not made by me... I'd like to delete it but I think it is used somewhere else unrelated to creating a work order

Reply:
if it's not used we should be able to delete it